### DISCUSSION TITLE
Add a new style to select

### DISCUSSION THREAD

**Discussion by AminAliH47** [saadeghi/daisyui#1769] [Ideas / feature requests]
Please add a new style for selects like [Material UI selects](https://mui.com/material-ui/react-select/).
So that when the drop-down is opened, its style is not the browser's default.

**Comment by devopg** [comment]
Hello daisyUI is style framework. For your examples require JS.
Daisy UI will update select then new `<selectmenu>` tag will be suppoer all main browsers
